### PR TITLE
SLO List widget

### PR DESCRIPTION
**Pull request by kaixintai**
### What does this PR do?

Add docs for the new SLO List widget

### Motivation

https://docs-staging.datadoghq.com/kai/slo-list-widget/dashboards/widgets/slo_list/

### Additional Notes

The SLO List and Summary widgets are also supported for scheduled dashboard reporting.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by urseberry** [comment]
Docs editorial review on DOCS-4149

File: content/en/dashboards/widgets/slo_list.md
Comment (alai97) [inline content/en/dashboards/widgets/slo_list.md:18]:
```suggestion
{{< img src="dashboards/widgets/slo_list/slo_list_editor.png" alt="A search query defining the service as web-store in the SLO List widget editor" style="width:90%;" >}}
```

File: content/en/dashboards/widgets/slo_list.md
Comment (alai97) [inline content/en/dashboards/widgets/slo_list.md:14]:
```suggestion
{{< img src="dashboards/widgets/slo_list/slo_list_widget.png" alt="The SLO List widget displaying a list of SLOs" style="width:90%;" >}}
```

File: content/en/dashboards/widgets/slo_list.md
Comment (alai97) [inline content/en/dashboards/widgets/slo_list.md:23]:
```suggestion
2. Use tags to filter the list of SLOs (such as `service:foo, env:prod`). Template variables are supported. 
```

File: content/en/dashboards/widgets/slo_list.md
Comment (alai97) [inline content/en/dashboards/widgets/slo_list.md:24]:
```suggestion
3. Choose the maximum number of SLOs to display (the default is 100). The most recently created SLOs appear at the top of the list. 
```

File: content/en/dashboards/widgets/slo_list.md
Comment (alai97) [inline content/en/dashboards/widgets/slo_list.md:25]:
```suggestion
4. Optionally, give the widget a title.

When you are ready to create the widget, click **Save**.
```

File: content/en/dashboards/widgets/slo_list.md
Comment (alai97) [inline content/en/dashboards/widgets/slo_list.md:9]:
```suggestion
  text: "Track the status of all your SLOs in Datadog"
- link: "/dashboards/graphing_json/"
  tag: "Documentation"
  text: "Building Dashboards using JSON" 
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some copy suggestions!

**Review by kaixintai — COMMENTED** [review]
Ok! I applied your changes, I hope I got all of them. Can you help me merge this once it's approved? @alai97

**Review by alai97 — APPROVED** [review]

**Pull request merged**
